usr/local/www/diag_dns.php

```diff
@@ -170,116 +174,142 @@ function display_host_results ($address,$hostname,$dns_speeds) {
 }
 
 include("head.inc"); ?>
-<body link="#0000CC" vlink="#0000CC" alink="#0000CC">
-<?php include("fbegin.inc"); ?>
-<table width="100%" border="0" cellpadding="0" cellspacing="0" summary="diag dns">
-        <tr>
-                <td>
+
 <?php if ($input_errors) print_input_errors($input_errors); ?>
 	<form action="diag_dns.php" method="post" name="iform" id="iform">
-	  <table width="100%" border="0" cellpadding="6" cellspacing="0" summary="tabcont">
-		<tr>
-			<td colspan="2" valign="top" class="listtopic"> <?=gettext("Resolve DNS hostname or IP");?></td>
-		</tr>
-        <tr>
-		  <td width="22%" valign="top" class="vncellreq"><?=gettext("Hostname or IP");?></td>
-		  <td width="78%" class="vtable">
-            <?=$mandfldhtml;?>
-			<table summary="results">
-				<tr><td valign="top">
-			<input name="host" type="text" class="formfld" id="host" size="20" value="<?=htmlspecialchars($host);?>" />
-			</td>
-			<?php if ($resolved && $type) { ?>
-			<td valign="middle">&nbsp;=&nbsp;</td><td>
-			<font size="+1">
+
+<!-- First table displayes the DNS lookup form and hte resulting IP addresses -->
+
+	<div class="table-responsive">
+		<table width="100%" class="table" summary="results">
+		<thead>
+			<tr>
+				<th width="30%">Host name</th>
+				<?php if ($resolved && $type) { ?> 
+				<th>IP Addresse(s)</th>
+				<?php } ?>
+			</tr>
+		</thead>
+		<tbody>
+			<tr class="active">
+				<td valign="top">
```

Comment:
Sure. I have not cleaned up the old code yet. Just looking for comments on the screen appearance and functionality at this point.

> On Mar 26, 2015, at 4:53 PM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_dns.php <https://github.com/SjonHortensius/pfsense/pull/42#discussion_r27254844>:
> 
> > +
> > +<!-- First table displayes the DNS lookup form and hte resulting IP addresses -->
> > +
> > +	<div class="table-responsive">
> > +		<table width="100%" class="table" summary="results">
> > +		<thead>
> > +			<tr>
> > +				<th width="30%">Host name</th>
> > +				<?php if ($resolved && $type) { ?> 
> > +				<th>IP Addresse(s)</th>
> > +				<?php } ?>
> > +			</tr>
> > +		</thead>
> > +		<tbody>
> > +			<tr class="active">
> > +				<td valign="top">
> valign definetly should not be here
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/42/files#r27254844>.
>